docs/reports/2022-07-21.md

```diff
@@ -0,0 +1,34 @@
+# 21st July 2022 
+
+## What we did this Week:
+### Friday 15th to Thursday 21st July 2022
+* Add NHS users to CKAN
+* Bug: CSV download button produces base64 encoded file
+* Build status filter
```

Comment:
What's this one?

Reply:
https://standardsportal.atlassian.net/browse/STAN-800

Reply:
Do you want me to take it out.